src/typeahead/typeahead.js

```diff
@@ -202,7 +202,7 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.debounce', 'ui.bootstrap
       return false;
     };
 
-    var getMatchesAsync = function(inputValue) {
+    var getMatchesAsync = function(inputValue, evt) {
```

Comment:
I think it is fine - only thing is to perhaps add a note in the documentation that this can be the case.